typesafe_conductr_cli/conduct.py

```diff
@@ -7,14 +7,18 @@
 import os
 
 
+defaultHost = os.getenv('HOSTNAME', '127.0.0.1')
```

Comment:
Yes. I think I have used camelCase somewhere else also. Will check all the files and replace to underscore.